README.md

```diff
@@ -229,6 +232,15 @@ Number of people named "Alice": 1
 Alice
 ```
 
+Read-only and best effort queries can be called by passing ```options``` to ```Txn.query``` and ```Txn.queryWithVars```. For example:
+
+```js
+const options = { readOnly: true, bestEffort: true };
+const res = await dgraphClient.newTxn().query(query, options);
```

Comment:
dgraph-js uses options on transaction, not on a query inside transaction. That seems logical, can it be done here?

https://github.com/dgraph-io/dgraph-js/commit/65d45150cf338a2126960e71179d6c9c3c6c4d51

Reply:
Sure, I can move that to the transaction

Reply:
Done